packages/block-library/src/paragraph/block.json

```diff
@@ -64,6 +63,7 @@
 				"fontSize": true
 			}
 		},
+		"__experimentalBlockBindings": true,
```

Comment:
If I remember correctly, I just kept that attribute because the partially synced patterns were using it. From [the second pull request](https://github.com/WordPress/gutenberg/pull/57258), it shouldn't be needed for anything else.

It'd be good to double check that anyway.

Reply:
As long as all blocks and attributes allowed are locked in the code, we can omit block supports from the initial scope. We will definitely need it later. Maybe, it’s going to be needed for V1 as a way for extenders to opt out from the feature. It’s a separate discussion to have that shouldn’t block the progress on the prototype.

Reply:
Ok I removed this and also [removed the block supports requirement](https://github.com/WordPress/gutenberg/pull/57249/commits/7fb646cae16f1703cfdee25a7f8629e6b020fec3) for the partially synced patterns.